README.md

```diff
@@ -151,6 +151,7 @@ This is the result from the auth server:
 - **scopes** - ([`string`]) the scopes the user has agreed to be granted
 - **authorizationCode** - (`string`) the authorization code (only if `skipCodeExchange=true`)
 - **codeVerifier** - (`string`) the codeVerifier value used for the PKCE exchange (only if both `skipCodeExchange=true` and `usePKCE=true`)
+- **connectionTimeoutSeconds** - (`number`) configure the request timeout interval in seconds.
```

Comment:
Would be worth clarifying here that this needs to be a `positive and non-zero` (since looks like we're ignoring zeros). Also what's the default for both platforms if this isn't set.

Reply:
Ah it would probably make more sense to reuse the platform/native lib defaults here. 

AppAuth-Android [sets a default of 15 seconds](https://github.com/openid/AppAuth-Android/blob/master/library/java/net/openid/appauth/connectivity/DefaultConnectionBuilder.java#L38). AppAuth-iOS doesn't set this value so [the default is 60 seconds](https://developer.apple.com/documentation/foundation/urlsessionconfiguration/1408259-timeoutintervalforrequest/).